sbi/neural_nets/density_estimators/mixed_density_estimator.py

```diff
@@ -37,6 +35,7 @@ def __init__(
                 evaluated (and which is also the event_shape of samples).
             condition_shape: Shape of the condition. If not provided, it will assume a
                 1D input.
+            embedding_net: Embedding net for the continuous part of the condition.
```

Comment:
I think it's fine to just say "condition" here (without the "continuous part"), no?